src/App.js

```diff
@@ -26,6 +26,9 @@ function App() {
       }
       <Router history={history}>
         <Switch>
+          <Route exact path="/">
+            <Redirect to="/login"/>
+          </Route>
```

Comment:
met uniquement <Redirect from="/" to="/login" /> pas besoin du Router. Et place le à la fin de toute les routes.
Sinon en étant connecté et en mettant la racine on se fait déconnecter

Reply:
Merci car j'avais tester dès le début la syntaxe que tu m'avais passer ` <Redirect from="/" to="/login"/>` mais du coup   la redirection fonctionne mais la page du login ne marcher pas ..du coup c'était pour ça que j'avais fait cette syntaxe ! grâce à ton commentaire la syntaxe que tu ma passer et en le plaçant à la fin , là ça marche 👍  Merci !